app/src/test/java/com/dicoding/picodiploma/sejiwaproject/features/match/search/SearchMatchPresenterTest.kt

```diff
@@ -1,5 +1,62 @@
 package com.dicoding.picodiploma.sejiwaproject.features.match.search
 
-import org.junit.Assert.*
+import com.dicoding.picodiploma.sejiwaproject.commons.api.ApiRepository
+import com.dicoding.picodiploma.sejiwaproject.features.TestContextProvider
+import com.dicoding.picodiploma.sejiwaproject.features.match.search.model.SearchMatch
+import com.dicoding.picodiploma.sejiwaproject.features.match.search.model.SearchMatchResponse
+import com.google.gson.Gson
+import kotlinx.coroutines.Deferred
+import kotlinx.coroutines.runBlocking
+import org.junit.Before
+import org.junit.Test
+import org.mockito.ArgumentMatchers
+import org.mockito.Mock
+import org.mockito.Mockito
+import org.mockito.MockitoAnnotations
 
-class SearchMatchPresenterTest
\ No newline at end of file
+class SearchMatchPresenterTest{
+
+    @Mock
+    private lateinit var view: SearchMatchView
+
+    @Mock
+    private lateinit var gson: Gson
+
+    @Mock
+    private lateinit var apiRepository: ApiRepository
+
+    @Mock
+    private lateinit var apiResponse: Deferred<String>
+
+    private lateinit var presenter: SearchMatchPresenter
+
+    @Before
+    fun setUp() {
+        MockitoAnnotations.initMocks(this)
+        presenter = SearchMatchPresenter(view, apiRepository, gson, TestContextProvider())
+    }
+
+    @Test
+    fun `test get search match is ready`() {
```

Comment:
This test case is more suitable if the name is _**test get search match is success**_.

Reply:
resolved, my bad.